src/proc/y16_10msb-to-y16.cpp

```diff
@@ -0,0 +1,36 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#include "y16_10msb-to-y16.h"
```

Comment:
the file's name is weird: you have half "_" and half "-".
Besides that, you chose to rename to y16i_10msb_to_y16y16 - so one "y16 is missing in the file's name.

Reply:
Fixed the `_`  in the file name.
There are two files, existing that converts interleaved y16**i** to two y16 streams and was renamed. Plus a new file that handles one y16 stream with 10 bit data and scales it to 16 bit data.